### PR TITLE
Publicity ui

### PR DESCRIPTION
**Pull request by Arushi1912**
### Description
Adding publicity permissions feature in the portal.

### Type of Change:

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have performed a self-review of my own code or materials
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by bhawesh96** [comment]
Rebase master and squash your commits

File: src/Services/firebaseStorageService.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ export const uploadProfilePic = (uid, file, callback) => {
     })
 }
 
+export const uploadPoster = (uid, file, callback) => {
+
+}
```
Comment (bhawesh96) [inline src/Services/firebaseStorageService.js]: Missing code

File: src/components/ClassroomComponent/BookingComponent/Rooms.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ class RoomsContainer extends Component {
     this.state = {
       selectedRooms: this.props && this.props.selectedRooms,
       takenRooms: this.props && this.props.takenRooms,
-      bookedRooms: [3202,3205]
+      bookedRooms: [3202, 3205]
```
Comment (bhawesh96) [inline src/components/ClassroomComponent/BookingComponent/Rooms.js]: This change shouldn't be in this commit

**Comment by bhawesh96** [comment]
@Arushi1912 
Please revert unrequired files like `BookingComponent.js`, `Rooms.js`. Please check `status` and/or `diff` before commiting or while raising a PR.

**Comment by rsarky** [comment]
Could you change the drawer icon for `Publicity Permission`, it's the same as `My Events` rn

**Comment by rsarky** [comment]
Heyy, could you resolve the merge conflicts?

**Comment by Arushi1912** [comment]
Hii. Check the PR once!

**Comment by rsarky** [comment]
Hey, I checked this out
- change icon for publicity requests
- Specifying number of infodesks should be mandatory? (if infodesk option is selected)
- Uploading the poster should be mandatory? (if poster option is selected)
- For authorities viewing the publicity request, we probably need to change the pdf that we render when you click on view. Right now most of the fields are empty.

**Comment by Arushi1912** [comment]
1. what icon are you suggesting?
2 and 3. okay i'll get to it. 
4. that'll be done from the backend right?

**Comment by rsarky** [comment]
For the last point basically we need to display information consistent with publicity requests so that the authorities can review them right? Eg. For Posters we will have to display the poster along with the places the poster will be displayed at. So on and so forth for the other categories. Need to discuss about the design a bit on this

**Comment by Arushi1912** [comment]
Wouldn't that be similar to the view publicity dialog? Yeah we'll discuss it then

**Comment by Arushi1912** [comment]
Hey can you check out the recent changes?

**Comment by rsarky** [comment]
Heyy, This looks good for now.
Merging to master. 
Can we track this
> For authorities viewing the publicity request, we probably need to change the pdf that we render when you click on view. Right now most of the fields are empty.

As a separate issue for the time being?

**Pull request merged**